src/modules/billing/mappers/transaction.js

```diff
@@ -56,6 +57,9 @@ const dbToModel = row => {
     volume: row.volume ? parseFloat(row.volume) : null,
     agreements: mapDBToAgreements(row)
   });
+  if (row.billingVolume) {
```

Comment:
If row.billingVolumes had been left as an array, would it be possible find the correct one by looking at the transaction season and the financial year of its charge period? 

I think the repo should probably return the data with relationships that match the DB schema, and the mapping to a single billingVolume should take place here